### ISSUE TITLE
Websocket stuck

### ISSUE THREAD

**Issue by idoall** [thrasher-corp/gocryptotrader#901]
# New Issue

Websocket stuck

When switching to a usable proxy, the websocket will get stuck when trying connect again.
## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System:
mac 11.5
* GoCryptoTrader version (`gocryptotrader -version`):
GoCryptoTrader v0.1 amd64 go1.17.6 pre-release.
## Current Behavior
```
When the proxy network switches, websockt will output "binance websocket has been disconnected. Reason: read TCP 10.17.0.6:65306 - > 52.68.124.244:9443: use of closed network connection"
```

1. Use a timer to detect websocket every 5 minutes
2.
```
If ws.Isconnected() {
    return
}else{
    ws.connect()
}
```

### Failure Logs

```
binance websocket has been disconnected. Reason: read TCP 10.17.0.6:65306 - > 52.68.124.244:9443: use of closed network connection
```

**Comment by thrasher-** [comment]
Hey @idoall, are you still experiencing this issue after #1059 was merged in?

**Comment by thrasher-** [comment]
Closing this due to being stale but please feel free to re-open this if you're still experiencing an issue.
